src/pages/instances/forms/InstanceFileTypeSelector.tsx

```diff
@@ -0,0 +1,45 @@
+import { RadioInput } from "@canonical/react-components";
+import { FC } from "react";
+
+export type InstanceFileType = "instance-backup" | "external-format";
+
+interface Props {
+  value: InstanceFileType;
+  onChange: (value: InstanceFileType) => void;
+  hide?: boolean;
+}
+
+const InstanceFileTypeSelector: FC<Props> = ({ value, onChange, hide }) => {
+  if (hide) {
+    return null;
+  }
```

Comment:
You can do it this way, personally prefer this decision in the parent to keep the interface clean. Like `{isConditional && <InstanceFileTypeSelector />}`

Reply:
agreed, that's better :+1: